internify/src/components/organisms/styles/AuthNavbar.css

```diff
@@ -0,0 +1,62 @@
+.auth_navbar_container {
+  height: 80px;
+  padding-top: 20px;
+}
+
+.auth_navbar_link {
+  text-decoration: none;
+  color: black;
+}
+
+
+.auth_navbar_links {
+  display: flex;
+  flex-direction: row-reverse;
+  margin-top: -30px;
+}
```

Comment:
You have CSS classes that never refer to anything. You don't have a class name of `auth_navbar_links` in your component